.github/workflows/build-and-test.yml

```diff
@@ -53,4 +53,4 @@ jobs:
         uses: actions-rs/cargo@v1
         with:
           command: nextest
-          args: run --all-features --no-fail-fast --cargo-profile ci --test-threads "num-cpus" --retries 2
+          args: run --all-features --no-fail-fast --cargo-profile ci --test-threads "num-cpus" --retries 2 -p iota-sdk -p iota-sdk-bindings-core
```

Comment:
Do we want to do that also for private tangle tests ?

Reply:
Added

Reply:
Change the github aliases too?

Reply:
Done